app/assets/stylesheets/layouts/_notifications.scss

```diff
@@ -0,0 +1,54 @@
+$notification-radius: 5px;
```

Comment:
There are a few things in this file.

* See prior comment about where sass variables should live.
* This is a BEM component (no `.l-` prefix) and so belongs in `components` rather than `layouts`
* The file should be named according to the component's name, so `_global_alert.scss`
* It might be an idea to rename it entirely, to something local to the project. It's not really an extension of `global-alert` elsewhere in MAS, so it should probably have a different name to disambiguate.
* There are a few long selectors in this file. BEM ought to mean you only ever have to specify one class (in theory) — so it seems like you're maybe fighting specificity here? If so, can that be resolved in some other way? Renaming the component might help if it's frontend that's interfering.

There's a bit of repetition in here that could probably be extracted too but it's no big deal.